src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,79 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
```

Comment:
Ensure that `getUser` is a stable function reference. If it changes between renders, it should be included in the `useEffect` dependency array to prevent stale closures.